source/_integrations/myuplink.markdown

```diff
@@ -77,6 +77,27 @@ Common use cases include:
 - **Automation**: Adjust the temperature curve offset during holiday mode
 - **Analytics**: View long-term statistics and graphs for the relevant sensors
 
+## Example
+
+Automation that will send a notification to a smartphone when the hot water reserve is getting low. Note that actual entity name varies between models of heat pumps. You will have to adapt the yaml code to your own installation.
+
+```yaml
+automation:
+  alias: Notify on low hot water reserve
```

Comment:
Strings should be enquoted, see also our YAML style guide:

<https://developers.home-assistant.io/docs/documenting/yaml-style-guide>

This comments applies to multiple items in this example.